### PR TITLE
Add unit tests for consultations and mastery checks (add, view, delete, edit)

### LINKED ISSUE #99: Implement unit tests for adding consultations

### LINKED ISSUE #100: Implement unit tests for deleting consultations

### LINKED ISSUE #174: Implement unit tests for adding mastery checks

### LINKED ISSUE #175: Implement unit tests for deleting mastery checks

### LINKED ISSUE #306: Implement unit tests for editing mastery checks

### LINKED ISSUE #307: Implement unit tests for viewing mastery checks

### LINKED ISSUE #308: Implement unit tests for viewing consultations

### PR DESCRIPTION
**Pull request by wireseo**
Fixes #306 
Fixes #307 
Fixes #308 
Fixes #99 
Fixes #100 
Fixes #174 
Fixes #175

### CODE REVIEW

File: src/main/java/seedu/jarvis/model/consultation/ConsultationNameContainsKeywordsPredicate.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package seedu.jarvis.model.consultation;
```
Comment (peironggg) [inline src/main/java/seedu/jarvis/model/consultation/ConsultationNameContainsKeywordsPredicate.java:1]: Great abstraction for checking name against predicate!

File: src/test/java/seedu/jarvis/logic/commands/AddConsultationCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
+package seedu.jarvis.logic.commands;
```
Comment (peironggg) [inline src/test/java/seedu/jarvis/logic/commands/AddConsultationCommandTest.java:1]: Great coverage!

File: src/test/java/seedu/jarvis/logic/commands/AddMasteryCheckCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
+package seedu.jarvis.logic.commands;
```
Comment (peironggg) [inline src/test/java/seedu/jarvis/logic/commands/AddMasteryCheckCommandTest.java:1]: I like the naming of the test functions as well

File: src/test/java/seedu/jarvis/testutil/EditMasteryCheckDescriptorBuilder.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
+package seedu.jarvis.testutil;
```
Comment (peironggg) [inline src/test/java/seedu/jarvis/testutil/EditMasteryCheckDescriptorBuilder.java:1]: Fantastic amount of tests and helper classes added 👍

File: src/main/java/seedu/jarvis/model/masterycheck/MasteryCheckNameContainsKeywordsPredicate.java [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.jarvis.commons.util.StringUtil;
+
+/**
+ * Tests that a {@code MasteryCheck}'s student name matches any of the keywords given.
+ */
+public class MasteryCheckNameContainsKeywordsPredicate implements Predicate<MasteryCheck> {
```
Comment (ngzhenteng) [inline src/main/java/seedu/jarvis/model/masterycheck/MasteryCheckNameContainsKeywordsPredicate.java:11]: Great naming the class. now when any code calls MasteryCheckNameContainsKeywordsPredicate.test(mc) it makes a lot of sense!

**Review by ngzhenteng — COMMENTED** [review]
Thank you for writing so many tests! LGTM!

**Pull request merged**